corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -65,7 +65,20 @@
     <% if (endPage && pageCount > 1) { %>
       <div class="container">
         <div class="row">
-          <div class="col-sm-9">
+          <div class="col-sm-2 module-per-page-container">
```

Comment:
This looks big enough to be worth extracting as a template and sharing with `users.html`